workflows/hifiasm/trio_hifiasm.cohort.wdl

```diff
@@ -0,0 +1,66 @@
+version 1.0
+
+# This is a new workflow with new algorithm to independently run yak together with trio-level hifiasm, coded by Charlie Bi
+# A new algrothm using new data input (parents_list) together with fasta_info is implemented to make yak computation easy
+# This workflow is dependent on the output/fasta_info from running smrtcells
+#
+# This new workflow is to replace Venkat's sample_trio.trial.wdl oringally resided in /sample
+
+import "../common/structs.wdl"
+import "./tasks/yak.wdl" as yak
+import "./tasks/hifiasm_trio.wdl" as hifiasm_trio_assemble
+
+workflow trio_hifiasm_cohort {
+  input {
+    Array[Pair[String,Array[File]]] fasta_info
+    Array[Array[String]] person_parents_names
+    Array[String]  parents_list
+    String pb_conda_image
+    IndexedData reference
+    Boolean trioeval
+    Boolean triobin    
+  }
+
+  #consider all siblings having the same parents as often seen, can be generalized in the future
+  scatter (sample in fasta_info){
+    Array[File] parents_fasta_1 = if (sample.left == parents_list[0]) then sample.right else []
+    Array[File] parents_fasta_2 =  if (sample.left == parents_list[1]) then sample.right else []
+  }
+  
+  call yak.yak as person_yak1 {
+        input:
+          sample_name = parents_list[0],
+          movie_fasta = flatten(parents_fasta_1),
+          pb_conda_image = pb_conda_image
+  }
+
+  call yak.yak as person_yak2 {
+        input:
+          sample_name = parents_list[1],
```

Comment:
@cbi-star  shouldn't this be:

`parents_fasta_2`

Reply:
parents_list[1] is the second parents_name